src/data/showcases.js

```diff
@@ -1198,6 +1198,15 @@ export const Showcases = [
     source: null,
     tags: ["wallet", "nftsupport"],
   },
+  {
+    title: "DROPSPOT",
+    description:
+      "Cardano's innovative NFT marketplace is now live with premium NFT-as-a-Service, project advisory & design, minting services, white label  services, claim services, loyalty programs, collectible design & development, activations and artist collaborations.",
```

Comment:
```suggestion
      "Premium NFT-as-a-Service, project advisory & design, minting services, white label services, claim services, loyalty programs, collectible design & development, activations and artist collaborations.",
```
This wold be the longest project description we have.  What about trimming the promotional language like this, which would keep it down to just a "very long" (but straightforward) description?  Also the term `now` doesn't mean anything in the long term and the term`live` is already implied (as a requirement for all posted web links).